specs/core/0_beacon-chain.md

```diff
@@ -1100,18 +1101,24 @@ When `is_genesis_trigger(deposits, timestamp) is True` for the first time, let:
 
 * `genesis_deposits = deposits`
```

Comment:
> Before genesis has been triggered and whenever the deposit contract emits a `Deposit` log, call the function `is_genesis_trigger(deposits: Sequence[Deposit], timestamp: uint64) -> bool` where:

Update this to say to run on each `block` rather than each log